### PR TITLE
Update container-run.md

### PR DESCRIPTION
**Pull request by dbjpanda**
Added an example for executing command on running container.

### CODE REVIEW

File: docs/cookbook/container-run.md [hunk 1]
```diff
@@ -208,3 +208,44 @@ $hostConfig->setPortBindings($portMap);
...
+    $stdoutText .= $stdout;
+});
+
+$stream->onStderr(function ($stderr) use (&$stderrText) {
+    $stderrText .= $stderr;
+});
+
+$stream->wait();
+var_dump([ "stdout" => $stdoutText, "stderr" => $stderrText ]) ;
+```
```
Comment (mohammadreza-soft74) [inline docs/cookbook/container-run.md]: thanks a lot . i spent all day on this finally i found it here .

**Comment by dbjpanda** [comment]
@joelwurtz Can you merge it if it looks good for you.

**Comment by michaelradionov** [comment]
+

**Comment by michaelradionov** [comment]
please merge it! so helpful!

**Comment by mohammadreza-soft74** [comment]
how can i write to container STDIN on exec.
for example i want run python code that need input .
how can i give input?
thanks in advance

**Comment by mohammadreza-soft74** [comment]
@dbjpanda how can i write to container STDIN on exec.
for example i want run python code that need input .
how can i give input?
thanks in advance

**Comment by dbjpanda** [comment]
$execConfig->setCmd(['/run.py',$arguments]);
Did you try this ? Note run.py should be present there in your py container.

**Comment by mohammadreza-soft74** [comment]
@dbjpanda  yes i try it . but i want to interact with python app. 

$remoteSocket ='192.168.85.34:2375';
$ssl = false;

// Connect to Docker
$client = DockerClientFactory::create([
    'remote_socket' => $remoteSocket,
    'ssl' => $ssl
]);
$docker = Docker::create($client);

$config = new ContainersCreatePostBody();
$config->setTty(true);
$config->setOpenStdin(true);
$config->setAttachStderr(false);
$config->setAttachStdout(false);
$config->setAttachStdin(false);
$config->setStdinOnce(false);
$config->setImage("test:2");

 $creation = $docker->containerCreate($config);



$docker->containerStart($creation->getId());

$execConfig = new ContainersIdExecPostBody();
$execConfig->setTty(true);
$execConfig->setAttachStdout(true);
$execConfig->setAttachStderr(true);
$execConfig->setAttachStdin(true);
$execConfig->setCmd(['/bin/bash']);

$execid = $docker->containerExec($creation->getId(),$execConfig)->getId();
$execStartConfig = new ExecIdStartPostBody();
$execStartConfig->setDetach(false);

// Execute the command
$stream = $docker->execStart($execid,$execStartConfig);
//var_dump($stream);die();

// To see the output stream of the 'exec' command
$stdoutText = "";
$stderrText = "";
$stream->onStdout(function ($stdout) use (&$stdoutText) {
    echo $stdout;

});
$stream->onStderr(function ($stderr) use (&$stderrText) {
    print_r("err: ".$stderr."\n");
});


$stream->wait();


its my code . in this example i want to interact with bash . i get output from container but icant sent anything to container. 
i think i should do something with Tty !!!
thanks in advance.

**Comment by mohammadreza-soft74** [comment]
anybody does not help me? 
It is very vital this issue

**Comment by joelwurtz** [comment]
Thanks @dbjpanda it should be correct yes.

@mohammadreza-soft74 I don't think this is possible with current endpoints, it needs to use the HTTP hijacking way of doing thing from docker (there is no Websocket endpoint). And this is not supported in this library ATM.

**Pull request merged**